src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -250,6 +250,11 @@ export const selectTradeSlippagePercentageDecimal: Selector<ReduxState, string>
   },
 )
 
+export const selectTradeSlippagePercentage: Selector<ReduxState, string> = createSelector(
```

Comment:
We don't appear to use this guy anywhere.

Reply:
https://github.com/shapeshift/web/pull/5032/commits/c6a9d5507ea1690661c05e2078bb1ee909df3449